### ISSUE TITLE
[mtt] OOM on SRIOV setup

### ISSUE THREAD

**Issue by amaslenn** [openucx/ucx#3049]
Configuration
```
OMPI: 4.0.0rc5
MOFED: MLNX_OFED_LINUX-4.4-2.0.7.0
Module: hpcx-gcc (2018-11-19)
Test module: none
Nodes: polar-sriov x4 (ppn=6(x4), nodelist=hpc-polar-01-vm[001-002],hpc-polar-02-vm[001-002])
```

MTT log: http://hpcweb.lab.mtl.com/hpc/mtr_scrap/users/mtt/scratch/shmem/20181119_184519_27486_12519_hpc-polar-01-vm001/html/test_stdout_X5NYvx.txt

Cmd:
`oshrun -np 24 -mca btl self --bind-to core -mca oshmem_proc_group_cache_size 10000 -x UCX_NET_DEVICES=mlx5_0:1 -mca osc ucx -x UCX_IB_ADDR_TYPE=ib_global -x UCX_RC_VERBS_TM_ENABLE=n -x UCX_DC_VERBS_TM_ENABLE=n -mca sshmem mmap -mca spml_ucx_heap_reg_nb 0 -mca coll '^hcoll' -mca coll_hcoll_enable 0 -mca spml ucx -mca pml ucx -x UCX_TLS=rc_x --map-by node -mca spml_yoda_bml_alloc_threshold 0 -x SHMEM_SYMMETRIC_HEAP_SIZE=1024M /hpc/mtr_scrap/users/mtt/scratch/shmem/20181119_184519_27486_12519_hpc-polar-01-vm001/installs/DoBX/tests/mpp_27/hpc_tests.git/mpp_bench_v1.0/bin/all2all.shmem 0 4096 1024 3`

Output:
```sh
--------------------------------------------------------------------------
Primary job  terminated normally, but 1 process returned
a non-zero exit code. Per user-direction, the job has been aborted.
--------------------------------------------------------------------------
--------------------------------------------------------------------------
oshrun noticed that process rank 18 with PID 17927 on node hpc-polar-02-vm001 exited on signal 9 (Killed).
--------------------------------------------------------------------------
```

Free mem:
```sh
$ pdsh -w $SLURM_NODELIST 'free -g'
hpc-polar-02-vm002:               total        used        free      shared  buff/cache   available
hpc-polar-02-vm002: Mem:             15           5           7           0           2           7
hpc-polar-02-vm002: Swap:             1           0           1
hpc-polar-01-vm001:               total        used        free      shared  buff/cache   available
hpc-polar-01-vm001: Mem:             15           5           7           0           2           7
hpc-polar-01-vm001: Swap:             1           0           1
hpc-polar-02-vm001:               total        used        free      shared  buff/cache   available
hpc-polar-02-vm001: Mem:             15           6           6           0           3           5
hpc-polar-02-vm001: Swap:             1           0           1
hpc-polar-01-vm002:               total        used        free      shared  buff/cache   available
hpc-polar-01-vm002: Mem:             15           6           6           0           3           6
hpc-polar-01-vm002: Swap:             1           0           1

```

**Comment by yosefe** [comment]
pls reboot polars, and check free mem is increased. it's rdmacm memleak to be fixed with mofed 4.5

**Comment by amaslenn** [comment]
Reboot added ~5-6 GB on each host.
